serverless/knative-client.adoc

```diff
@@ -0,0 +1,42 @@
+[id="knative-client"]
+= Using Knative Client
+include::modules/common-attributes.adoc[]
+:context: knative-client
+
+toc::[]
+
+Knative Client (`kn`) is the Knative command line interface (CLI). The CLI exposes commands for managing your applications, as well as lower level tools to interact with components of {product-title}. With `kn`, you can create applications and manage {product-title} projects from the terminal. 
+
+Knative client does not have its own log in mechanism. To log in to `kn` you must install the {product-title} CLI and use the `oc` login. Installation options for the CLI vary depending on your operating system.
+
+
+// Installation
+include::modules/cli-install.adoc[leveloffset=+1]
+include::modules/serverless-installing-cli-linux.adoc[leveloffset=+2]
+// RPM install procedure commented out
+// include::modules/serverless-installing-cli-linux-rpm.adoc[leveloffset=+2]
+include::modules/serverless-installing-cli-macosx.adoc[leveloffset=+2]
+include::modules/serverless-installing-cli-windows.adoc[leveloffset=+2]
+
+// Log in
+include::modules/cli-logging-in.adoc[leveloffset=+1]
+include::modules/serverless-creating-kubeconfig-file.adoc[leveloffset=+1]
+include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+2]
```

Comment:
@joaedwar : lets remove the other ways for kubeconfig and have the oc login way only.